go/libkb/config.go

```diff
@@ -549,7 +549,18 @@ func (f *JSONConfigFile) GetVDebugSetting() string {
 func (f *JSONConfigFile) GetAutoFork() (bool, bool) {
 	return f.GetTopLevelBool("auto_fork")
 }
-func (f *JSONConfigFile) GetRememberPassphrase() (bool, bool) {
+func (f *JSONConfigFile) GetRememberPassphrase(username NormalizedUsername) (bool, bool) {
+	if username.IsNil() {
+		goto Fallback
```

Comment:
this goto is basically being an if, can you rearrange this?

Reply:
One day I will find a legitimate use case